core-java/src/main/java/eu/neverblink/jelly/core/RdfTerm.java

```diff
@@ -0,0 +1,515 @@
+package eu.neverblink.jelly.core;
+
+import eu.neverblink.jelly.core.proto.v1.RdfDefaultGraph;
+import eu.neverblink.jelly.core.proto.v1.RdfGraphEnd;
+import eu.neverblink.jelly.core.proto.v1.RdfGraphStart;
+import eu.neverblink.jelly.core.proto.v1.RdfIri;
+import eu.neverblink.jelly.core.proto.v1.RdfLiteral;
+import eu.neverblink.jelly.core.proto.v1.RdfQuad;
+import eu.neverblink.jelly.core.proto.v1.RdfTriple;
+
+public sealed interface RdfTerm {
```

Comment:
Missing javadocs in the entire file. Please add them for all important public stuff.